trio/tests/test_ssl.py

```diff
@@ -98,14 +100,20 @@ def ssl_echo_serve_sync(sock, *, expect_fail=False):
                     # respond in kind but it's legal for them to have already
                     # gone away.
                     exceptions = (BrokenPipeError, ssl.SSLZeroReturnError)
-                    # Under unclear conditions, CPython sometimes raises
-                    # SSLWantWriteError here. This is a bug (bpo-32219), but
-                    # it's not our bug, so ignore it.
-                    exceptions += (ssl.SSLWantWriteError,)
                     try:
                         wrapped.unwrap()
                     except exceptions:
                         pass
+                    except ssl.SSLWantWriteError:  # pragma: no cover
+                        # Under unclear conditions, CPython sometimes raises
+                        # SSLWantWriteError here. This is a bug (bpo-32219),
+                        # but it's not our bug, so ignore it.
```

Comment:
```suggestion
                        # but it's not our bug, so ignore it on 3.7
                        # Christian Heimes thinks it's fixed in recent
                        # python versions so we fail the test instead
```

Reply:
```suggestion
                        # but it's not our bug.  Christian Heimes thinks
                        # it's fixed in 'recent' CPython versions so we fail
                        # the test for those and ignore it for earlier
                        # versions.
```

I guess this should also be checking `sys.implementation` since we're ignoring a known CPython bug.

Reply:
There aren't any other 3.8 implementations yet

Reply:
Doesn't sway my opinion, sorry.  `:]`